python/datasources/manual_uploads.py

```diff
@@ -21,5 +21,9 @@ def write_to_bq(self, dataset, gcs_bucket, _):
         bucket_files = gcs_to_bq_util.list_bucket_files(gcs_bucket)
```

Comment:
Do the manual uploads go to a separate gcs bucket from the automated ones? And are the manual uploads all in the same bucket? If so, do we want to be processing all of them every time?

Reply:
Yeah we separate all manual uploads into their own bucket but good point. We probably don't want to upload all the files at the same time every time.

I opened [an issue](#159 ) to separate these out so we can do one file at a time.